src/routes/teams.js

```diff
@@ -1018,3 +1041,26 @@ function convertKeys(input, method) {
     }
     return output;
 }
+
+async function getMaxTeamInvites({ user, team, server }) {
+    const maxTeamInvitesRes = await server.app.events.emit(server.app.event.MAX_TEAM_INVITES, {
+        user
+    });
+    const maxTeamInvites = maxTeamInvitesRes
+        .filter(d => d.status === 'success')
+        .map(({ data }) => data.maxInvites)
+        .sort()
+        .slice(-1)[0];
```

Comment:
```suggestion
        .pop();
```
If you want the last item in the array you can use `Array.prototype.pop`. Even though it accomplishes the same, `Array.prototype.slice` will create a new array which is not needed.